### PR TITLE
Ec3

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

Provide tutorial on EC3 and move existing HTC and ECAS guides to the right place

----

**Related issue :**

### CODE REVIEW

**Review by glarocca — DISMISSED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ git:
 before_install:
   - gem install mdl
   - npm install postcss-cli
```
Comment (andrea-manzi) [inline .travis.yml:13]:
```suggestion
  - npm install postcss-cli@7.1.2
```

**Comment by andrea-manzi** [comment]
Given that you are documenting also new options for the eglici which are not yet available (PR https://github.com/EGI-Foundation/egicli/pull/17 is not merged yet) maybe we need first to complete that PR before including the related docs?

File: content/en/users/fedcloud/ec3/basics.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+id                                Name              enabled    site
+--------------------------------  ----------------  ---------  ------
+3a8e9d966e644405bf19b536adf7743d  vo.access.egi.eu  True       CESGA
+```
+
+Using the project id and the site name, you can create the authorisation files
+needed for ec3:
+
+``` {.sh}
+$ egicli endpoint ec3-auth --site CESGA --project-id 3a8e9d966e644405bf19b536adf7743d
```
Comment (andrea-manzi) [inline content/en/users/fedcloud/ec3/basics.md:107]:
```suggestion
$ egicli endpoint ec3 --site CESGA --project-id 3a8e9d966e644405bf19b536adf7743d
```

**Review by andrea-manzi — APPROVED** [review]
good to go!

**Pull request merged**